.scrutinizer.yml

```diff
@@ -38,10 +38,24 @@ tools:
     runs: 2
     timeout: 2400
 
+checks:
+    php:
+        excluded_dependencies:
```

Comment:
4 spaces instead of 2 used
below on build key to

Works on scrutinizer but some IDEs could begin to moan about